### ISSUE TITLE
Doesnt found EOSSDK-Win64-Shipping.dll on Steam Version of KH 3D

### ISSUE THREAD

**Issue by FernandoRApud** [TopazTK/KH-ReFixed#162]
### What Re:Fined version are you using? (versions other than listed are not supported)

v3.20

### In which game is the bug happening?  (games other than listed are not supported)

Kingdom Hearts Dream Drop Distance HD

### The bug & it's reproduction steps

Copy the .exe from the download, put on the steam folder of the root directory game (where the other .exes are) and open just clicking it (not via Steam), this will drop this error. Probably is wanting the dll that i understand is from Epic Games.

### What should've hapepned instead

The game should open normal.

### I did the following:

- [X] Make sure the Re:Fined .exe and OpenKH are up-to-date.
- [X] [See if the issue has been mentioned before](https://github.com/KH-ReFined/KH-ReFined/issues).

**Comment by TrollanKojima** [comment]
Can confirm - The Re:Fined EXE seems to be looking for a .dll that exists in the EGS version, but not the Steam version. Damn shame, as the game runs roughly about the same as the launch EGS version without it.

**Comment by sonowz** [comment]
The very same error occurs with KH1 Final Mix with v3.20.

**Comment by ckhericks** [comment]
same here for KH1 Final Mix, steam version, remix v3.20

Keep in mind I think this project is deprecated and was migrated to this repo:
https://github.com/KH-ReFreshed/KH-ReFreshed

However that repo seems to have issues disabled, so we can't create an issue for this bug.

**Comment by ckhericks** [comment]
I just checked the KH Refined discord, and there are announcements and latest details as of 48 hours ago on their #announcements text channel.

TLDR - This repo was originally written to support the Epic Game Store version, and doesn't yet support the Steam version. This is no ETA yet for when support will be added. 

Check over there for the full updates and details.

In fact they specifically asked that issues not be opened about this topic, so you should probably self-close if you can.

**Comment by sonowz** [comment]
@ckhericks Thanks for the information!

I also checked the discord, and it seems this repo will continue to serve with above Re:Freshed repo merged to it.
If I understood correctly, the devs are working on a patch to fix EGS version first (broken by update) then maybe work on Steam version.
I think the fix might take quite a long time, since this project seems to tackle even Assembly code 😿

**Comment by TopazTK** [comment]
Re:Fined is incompatbile with the latest releases. Please do not use it until an update is given.
Closing.
